stock_picking_package_preparation/model/stock_picking_package_preparation.py

```diff
@@ -138,14 +138,16 @@ def _compute_weight(self):
         package = self.package_id
         quant_model = self.env['stock.quant']
         package_model = self.env['stock.quant.package']
-        quants = quant_model.browse(package.get_content())
-        # weight of the products only
-        net_weight = sum(l.product_id.weight * l.qty for l in quants)
-        # weight of the empty packages + products
-        child_packages = package_model.search([('id', 'child_of', package.id)])
-        weight = sum(child_packages.mapped('ul_id.weight')) + net_weight
-        self.net_weight = net_weight
-        self.weight = weight
+        if package:
```

Comment:
I usually prefer to do:
```python
if not package:
    return
[...]
````

It has 2 benefits here:
* it avoids an additional indentation level
* it reduces the diff to only 2 lines instead of a dozen